src/components/ArticlesById.jsx

```diff
@@ -1,5 +1,30 @@
+import SingleArticleCard from "./SingleArticleCard"
+import { useEffect, useState } from "react"
+import { fetchArticleById } from "../API/api"
+import { useParams } from "react-router-dom"
+
 const ArticlesById = () => {
-    return <div>ArticleByIdPage</div>
+    const [article, setArticle] = useState({})
+    const {article_id} = useParams()
+    const [isLoading, setIsLoading] =useState(true)
+    // console.log(article_id);
```

Comment:
remove console logs

Reply:
resolved the rouge CL and  will move buttons above the headline article :)